.github/actions/test-ruby/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Ruby Unit Test
+description: Ruby Unit Test report
+
+
+inputs:
+  RUBY_VERSION:
+    required: false
+    default: "2.7.4"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Ruby and install gems
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+        version: ${{ inputs.RUBY_VERSION }}
+
+    - name: Run Tests
+      run: |
+        cp .env.example .env
+        bundle install --path ./bundle --without production,development > /dev/null
+        bundle exec rake db:test:prepare > /dev/null
+        bundle exec rake core:db:test:prepare > /dev/null
+        bundle exec rake messaging:db:test:prepare > /dev/null
```

Comment:
For Rails projects other than prime-rails that don't have a core or messaging database, will these lines break the pipeline?